assets/constants.ts

```diff
@@ -5175,6 +5175,7 @@ export = {
   },
 
   "ENABLE_PREREQUISITE_SKILLS": false,
+  "ENABLE_LITERALLYCANVAS_EDITOR": false,
```

Comment:
Should this be edited as follows?
```suggestion
  "ENABLE_LITERALLY_CANVAS_EDITOR": false,
```

Reply:
Done.